src/TestUtils/src/Microsoft.Maui.IntegrationTests/BaseBuildTest.cs

```diff
@@ -15,7 +16,7 @@ public class BaseBuildTest
 		public const string MauiVersionCurrent = "8.0.0-rc.1.9171"; // this should not be the same as the last release
 		public const string MauiVersionPrevious = "7.0.86"; // this should not be the same version as the default. aka: MicrosoftMauiPreviousDotNetReleasedVersion in eng/Versions.props
 
-		char[] invalidChars = { '{', '}', '(', ')', '$', ':', ';', '\"', '\'', ',', '=', '.', '-', };
+		char[] invalidChars = { '{', '}', '(', ')', '$', ':', ';', '\"', '\'', ',', '=', '.', '-', ' ', };
```

Comment:
This was to work around a bug that I need to investigate further. Once I know more I will file a bug about spaces in certain paths. The bug is unrelated to this change, but this change runs into the bug and causes tests to fail.